recipe/bld.bat

```diff
@@ -5,9 +5,8 @@ set GDAL_VERSION=%%F
 if errorlevel 1 exit 1
 echo "set GDAL_VERSION=%GDAL_VERSION%"
 
+set GDAL_INCLUDE_PATH=%LIBRARY_INC%
+set GDAL_LIBRARY_PATH=%LIBRARY_LIB%
+
 "%PYTHON%" -m pip install --no-use-pep517 -vv . ^
-                          --global-option=build_ext ^
-                          --global-option="-I%LIBRARY_INC%" ^
-                          --global-option="-L%LIBRARY_LIB%" ^
-                          --global-option="-lgdal_i"
```

Comment:
Perhaps this line is still needed, given that the Windows builds are all failing?  I recall that `gdal_i` is a Windows-specfic identifer for the GDAL shared lib.

Reply:
In _theory_ I think it shouldn't be needed, since we have handling for gdal_i in our setup.py, but if I don't get it working, we can certainly revert to what was there before